rubicon/objc/objc.py

```diff
@@ -1644,3 +1644,23 @@ def finalize(self, cmd) -> None:
             anObject = get_instance_variable(self, 'observed_object', objc_id)
             ObjCInstance._cached_objects.pop(anObject, None)
             send_super(self, 'finalize')
+
+
+class ObjCBlockStruct(Structure):
+    _fields_ = [
+        ('isa', c_void_p),
+        ('flags', c_int),
+        ('reserved', c_int),
+        ('invoke', CFUNCTYPE(c_void_p, c_void_p)),
+    ]
+
+
+class ObjCBlock:
```

Comment:
Would it be possible to extend `ObjCInstance` here? That way the `ObjCBlock` object can be passed into methods, and can have methods called on it (which is useful for `retain`, `release` and `copy`).